docs/protocol_speficiation/components.md

```diff
@@ -0,0 +1,44 @@
+# List of components
+
+### Node identities and Mana
+
+Nodes receive rewards for being good actors in a system that secures the network and provides Sybil protection. 
+
+For more details see the [Goshimmer mana specs](./001-mana.md).
+
+### Message layout
+
+The concept of payloads creates infinite possibilities while also improving data processing efficiency. 
+
+### Autopeering
+
+A mechanism for automatically finding neighbors to build a robust ever-changing network.
+ 
+### Timestamps
+
+A declared time variable for messages that allows message ordering, rate control and identifies malicious behavior. 
+
+### Tip selection
+
+Faster, more secure and simpler. The new tip selection is not part of consensus anymore, but it allows us to reach finality within seconds.
+
+### Rate and congestion control
+
+Keeps spammers from harming the network and allocates bandwidth during congested times. 
+
+### Ledger state
+
+The extended Unspent Transaction Output (UTXO) model with the Parallel Reality Ledger State enables nodes to efficiently keep track and update all the balances while managing potential conflicts.
+
+### Consensus
+
+Decentralized random number generator to provide security, fast voting algorithm to provide quick conflict resolutions  and effective finality criteria to fit all applications needs. 
```

Comment:
Maybe we should be less general and mention FPC and FCoB?

Reply:
ok done